### PR TITLE
Added Java Math Methods: .floor()

### PR DESCRIPTION
**Pull request by itsvipindev**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Dhanunjay18** [comment]
i can solve this efficiently

**Comment by Dusch4593** [comment]
@Dhanunjay18 The issue for this particular method has already been assigned and you are commenting in the PR created for it. Please refer to the list of [issues that haven't yet been assigned to anyone](https://github.com/Codecademy/docs/issues?q=is%3Aissue+is%3Aopen+no%3Aassignee).

**Comment by Dusch4593** [comment]
@itsvipindev Let's make sure to sign the [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=1077) and check off the boxes in the "Checklist" near the top.

File: content/java/concepts/math-methods/terms/floor/floor.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+Title: '.floor()'
+Description: 'Returns the double value that is less than or equal to the argument and is equal to the nearest mathematical integer.'
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/floor/floor.md:3]:
```suggestion
Description: 'Returns the largest integer value that is less than or equal to the argument.'
```

File: content/java/concepts/math-methods/terms/floor/floor.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Tags:
+  - 'Methods'
+  - 'Functions'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.floor()`** method returns a  `integer` value that is less than or equal to argument and is equal to the nearest mathematical integer.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/floor/floor.md:15]:
```suggestion
The **`Math.floor()`** method returns the largest  `integer` value that is less than or equal to argument.
```

The nearest integer is not the same as the integer less than the argument.

File: content/java/concepts/math-methods/terms/floor/floor.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+## Syntax
+
+```pseudo
+Math.floot()
+```
+
+The `.floor()` method takes `integer` as an argument whose floor value is to be determined.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/floor/floor.md:23]:
```suggestion
The `.floor()` method takes a `double` or `float` as an argument whose floor value is to be determined.
```

File: content/java/concepts/math-methods/terms/floor/floor.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## Example
+
+The following example demonstrates using  of `java.lang.Math.floor()` method. 
+:
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/floor/floor.md:28]:
```suggestion
```

File: content/java/concepts/math-methods/terms/floor/floor.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+```pseudo
+Math.floot()
+```
+
+The `.floor()` method takes `integer` as an argument whose floor value is to be determined.
+
+## Example
+
+The following example demonstrates using  of `java.lang.Math.floor()` method.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/floor/floor.md:27]:
```suggestion
The following example demonstrates using the `.floor()` method:
```

File: content/java/concepts/math-methods/terms/floor/floor.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        System.out.println(Math.floor(d)); 
+  
+        // Input -2.3, Output -3.0 
+        // Nearest Integer(-3.0) < less than (-2.3) 
+        System.out.println(Math.floor(e)); 
+    } 
+}
+```
+
+The output will be a `integer` type
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/floor/floor.md:58]:
```suggestion
The output will be a `integer` type:

```

**Review by SSwiniarski — COMMENTED** [review]
@itsvipindev, I did a first review and left a few comments.

**Pull request closed without merging**